examples/backlist_both_tokens.py

```diff
@@ -0,0 +1,77 @@
+from flask import Flask, request, jsonify
+
+from flask_jwt_extended import (
+    jwt_refresh_token_required, jwt_required, create_access_token, get_jti,
+    create_refresh_token, get_jwt_identity, get_raw_jwt, JWTManager
+)
+
+
+app = Flask(__name__)
+
+# Enable blacklisting and specify what kind of tokens to check
+# against the blacklist
+app.config['JWT_SECRET_KEY'] = 'super-secret'  # Change this!
+app.config['JWT_BLACKLIST_ENABLED'] = True
+app.config['JWT_BLACKLIST_TOKEN_CHECKS'] = ['access', 'refresh']
+jwt = JWTManager(app)
+
+blacklist = set()
+
+
+@jwt.token_in_blacklist_loader
+def check_if_token_in_blacklist(decrypted_token):
+    jti = decrypted_token['jti']
+    return jti in blacklist
+
+
+# Endpoint for logging in
+@app.route('/login', methods=['POST'])
+def login():
+    username = request.json.get('username', None)
+    password = request.json.get('password', None)
+    if username != 'test' or password != 'test':
+        return jsonify({"msg": "Bad username or password"}), 401
+
+    # use a dict for identity, so that we can link the access token with
+    # the refresh token as a user claim
+    user = {"username": username}
+    refresh_token = create_refresh_token(identity=user)
+
+    # Embed the refresh token's jti in the access_token
+    user["refresh_jti"] = get_jti(refresh_token)
```

Comment:
I don't really like having having this data stored in what will become the `current_user` for the JWT. It complicates things, especially when using the complex objects from token and vice versa features in this extension. Could we use the `@jwt.user_claims_loader` instead to put the data into the JWT, then access it via `get_jwt_claims()` instead of `current_user` bellow?